procedures/foreman/branch.md.erb

```diff
@@ -59,7 +59,7 @@
   - [ ] Create [releases/<%= release %>/RPM-GPG-KEY-foreman](https://yum.theforeman.org/releases/<%= release %>/RPM-GPG-KEY-foreman) on yum.theforeman.org
   - [ ] Commit the [settings file](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/foreman/<%= release %>/settings) to the `theforeman-rel-eng` repository
 - [ ] Create new settings files for [client](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/client/<%= release %>/settings), ensure it has the rights `OSES` list and commit it too.
-- [ ] Create `configs/foreman/<%= release %>.yaml` based on the nightly and previous release. Ensure that any `based_on` entries are filled out and pointing to nightly version of tag.
+- [ ] Create `configs/foreman/<%= release %>.yaml` based on the nightly and previous release. Ensure that any `based_on` entries are filled out and pointing to nightly version of tag. The key here needs to be replaced by the 'short' (last eight digits) of the public gpg key. 
```

Comment:
```suggestion
- [ ] Create `configs/foreman/<%= release %>.yaml` based on the nightly and previous release. Ensure that any `based_on` entries are filled out and pointing to nightly version of tag. The key here needs to be replaced by the 'short' (last eight characters) of the public gpg key. 
```

I would note (and I thought I did before) that we have a GPGKEY variable in rel-eng tools. It may be better to create a small script that prints it and recommend using that.

https://github.com/theforeman/theforeman-rel-eng/blob/97cd19145819cdb3c053cb9c38a2fe6873ca098d/settings#L67-L68

Reply:
Since I'm revisiting this I opened what I intended: https://github.com/theforeman/theforeman-rel-eng/pull/202. I'm going to propose further changes.